tools/lint/main_ni.c

```diff
@@ -944,6 +962,7 @@ main_ni(int argc, char *argv[])
 cleanup:
     /* cleanup */
     erase_context(&c);
+    free(features_output);
```

Comment:
I think there is missing to release `set` when `generate_features_output` fails.